src/server/index.js

```diff
@@ -64,7 +65,9 @@ if (program.staticDir) {
 // Build the webpack configuration using the `baseConfig`
 // custom `.babelrc` file and `webpack.config.js` files
 const configDir = program.configDir || './.storybook';
+const dbPath = path.resolve(configDir, 'datastore.json');
```

Comment:
For now, we should disable this by default.
We should enable saving data, if a user asked to do that via an option.

We can change that later, but until we figure things out this needs to be opt-in.